src/averageAges.js

```diff
@@ -14,12 +14,15 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = people.filter((men) => men.sex === 'm'
+    && (century === undefined || Math.ceil(men.died / 100) === century));
```

Comment:
Don't compare with undefined/null or empty string. Use the logical not to check the falsy values, `!century`.